lexers/e/eiffel.go

```diff
@@ -0,0 +1,19 @@
+package e
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Eiffel lexer.
+var Eiffel = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "eiffel",
```

Comment:
This is upper case in pygments
```suggestion
		Name:      "Eiffel",
```

Reply:
tks